### ISSUE TITLE
Tracer extension leads to PHPUnit 5.7.27 tests failing

### ISSUE THREAD

**Issue by slashr** [DataDog/dd-trace-php#568]
Hi, 

I am trying to install datadog-php-tracer_0.30.1_amd64.deb inside our PHP application container. While the extension installs correctly, our PHPUnit test suite reports the following error and the build fails.

```
/usr/local/bin/phpdbg -dmemory_limit=4096M -qrr ./vendor/bin/phpunit --debug --configuration=app/phpunit.integration-frontend.xml --coverage-clover build/logs/coverage.xml --exclude-group=admin,realdata,outdated
[PHP Notice:  Undefined index: REQUEST_METHOD in /opt/datadog-php/dd-trace-sources/src/DDTrace/Bootstrap.php on line 108]
[PHP Notice:  Undefined index: REQUEST_URI in /opt/datadog-php/dd-trace-sources/src/DDTrace/Bootstrap.php on line 109]
PHPUnit 5.7.27 by Sebastian Bergmann and contributors.


Starting test 'Econa\Bundle\ApiBundle\Tests\Handler\Converter\ParamConverterTest::testParamConvertConstruct'.
./bin/devtest: line 22: 15377 Segmentation fault      (core dumped) APP_ENV=frontend_test /usr/local/bin/phpdbg -dmemory_limit=4096M -qrr ./vendor/bin/phpunit "--debug" "--configuration=${configuration}" "${extra_params[@]}"
command terminated with exit code 1
```
Is this a known issue?

**Comment by labbati** [comment]
Hi @slashr thanks for filing this issue. Never heard of this error. We will look into this as soon as possible. Would it be possible for you to share the core dump that was produced? 

If  you prefer to do it privately you can reach out to me `@luca.abbati` on our public slack `datadoghq.slack.com`.

**Comment by labbati** [comment]
Thanks @finfa for the investigation, we should be able to address that soon. In the meantime, as a workaround, you can set `DD_TRACE_ENABLED=false` in the shell where `phpunit` is executed.

**Comment by labbati** [comment]
@slashr @finfa just checking in. Did the workaround work for you?

**Comment by slashr** [comment]
@labbati Confirming that this workaround did not work for us.

```
[PHP Notice:  Undefined index: REQUEST_METHOD in /opt/datadog-php/dd-trace-sources/src/DDTrace/Bootstrap.php on line 108]
[PHP Notice:  Undefined index: REQUEST_URI in /opt/datadog-php/dd-trace-sources/src/DDTrace/Bootstrap.php on line 109]
```

**Comment by SammyK** [comment]
Hey @slashr & @finfa! We just released `0.34.1` which should fix this issue. Please let us know if you are still having issues after [upgrading](https://docs.datadoghq.com/tracing/setup/php/#upgrading).

**Comment by SammyK** [comment]
I'm closing this issue due to inactivity. Please feel free to reopen if you're still having issues.
